src/domains/checkout-app/05_checkout_fe.tf

```
@@ -157,7 +157,12 @@ module "checkout_cdn" {
       order = 5
 
       // conditions
-      url_path_conditions       = []
+      url_path_conditions = [{
+      operator         = "BeginsWith"
+      match_values     = ["/static/fonts/"]
```

```suggestion
      match_values     = ["/fonts/"]
```

same consideration from the below comment